### DISCUSSION TITLE
Could not load type 'Quartz.Plugin...

### DISCUSSION THREAD

**Discussion by fdtaljaard** [quartznet/quartznet#728] [General] [answered]
After upgrading Quarts to the latest version 3.0.7.0 I'm getting the following exception.

Could not load type 'Quartz.Plugin.History.LoggingJobHistoryPlugin' from assembly 'Quartz, Version=3.0.7.0, Culture=neutral, PublicKeyToken=f6b8c98a402cc8a4'.

My Code:

` var properties = new NameValueCollection
                {
                    ["quartz.plugin.triggHistory.type"] = "Quartz.Plugin.History.LoggingJobHistoryPlugin",
                    ["quartz.plugin.jobInitializer.type"] = "Quartz.Plugin.Xml.XMLSchedulingDataProcessorPlugin",
                    ["quartz.plugin.jobInitializer.fileNames"] = "~/quartz_jobs.xml",
                    ["quartz.plugin.jobInitializer.failOnFileNotFound"] = "true",
                    ["quartz.plugin.jobInitializer.scanInterval"] = "120"
                };
       
                // First we must get a reference to a scheduler
                _schedulerFactory = new StdSchedulerFactory(properties);
                _scheduler = _schedulerFactory.GetScheduler();`

**Comment by lahma** [comment, marked as answer]
You need to reference NuGet package [Quartz.Plugins](https://www.nuget.org/packages/Quartz.Plugins) and specify assembly qualified name for the plugin:

https://github.com/quartznet/quartznet/blob/a5f191d14f844bc29b2838e837c671d544cffb3b/src/Quartz.Examples/15_ConfigureJobSchedulingByUsingXmlConfigurations/ConfigureJobSchedulingByUsingXmlConfigurationsExample.cs#L45-L56

**Comment by fdtaljaard** [comment]
Thank you, I did reference the library but I noticed the namespace Plugin is Plugins now.
